crud/insert.lua

```diff
@@ -147,7 +147,12 @@ function insert.object(space_name, obj, opts)
     opts = opts or {}
 
     -- insert can fail if router uses outdated schema to flatten object
-    opts.add_space_schema_hash = true
+    opts = {
+        timeout = opts.timeout,
```

Comment:
If then original `opts` is nil then `opts.timeout`, `opts.bucket_id`, `opts.fields` are equal to `nil` too and adding them to table is useless.
If the original `opts` is not `nil` then we reassign original {timeout, bucket_id, fields} values to the same key...

Didn't get the idea behind it.

Reply:
Before this fix original opts table was modified by adding a key `add_space_schema_hash`. To avoid this modification we need to copy the values of original opts table, that's why I use this construction `opts = {timeout = opts.timeout, ...}` (there is another method: table.copy(opts), but I thought it that `opts` is not huge or complex table to perform `table.copy()`)